tools/run_parallel.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
I notice you created a new directory `tools` to store the script. Since it's testing related and called by Evergreen task, suggest to move the script to the existing `test/evergreen` directory.

Reply:
Hi @lukech , I wrote `run_parallel.sh`. I don't have a strong view on its location, but I put it in the tools directory as I deliberately created it to be reusable for multiple purposes, including testing of course.

Reply:
Alright. I'm not super strong on this suggestion. Feel it's a bit overkill to introduce a new top-level directory just for a single script (and `test/evergreen` is a natural alternative in this case).

Reply:
Hi @lukech , the `tools` directory was already there, I simply added the `run_parallel.sh` file to  it.

Reply:
Oops, sorry I thought it's a new directory. Never mind then.